### PR TITLE
fix qos profiles, must use ros1_bridge specific listener

### PR DESCRIPTION
**Pull request by dirk-thomas**

### CODE REVIEW

**Comment by wjwwood** [comment]
lgtm, but why does it need the sensor qos profile (presumably for unreliable, but why is that needed)?

**Comment by dirk-thomas** [comment]
Because a reliable (default qos) subscriber won't get any data from a best effort publisher.

File: README.md [hunk 1]
```diff
@@ -112,11 +112,12 @@ The ROS 1 node will start printing the published messages to the console.
 ---
 
 Now we start the ROS 2 listener.
+It is important to use the listener from the `ros1_bridge` package since it uses different QoS parameters then the "normal" `listener`.
```
Comment (esteve) [inline README.md]: then -> than

Comment (dirk-thomas) [inline README.md]: Fixed.

**Comment by esteve** [comment]
+1

**Pull request merged**